int/ks-devops-ext-tekton-common/config/config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "app": {
+    "name": "common-tekton-manifests",
```

Comment:
I prefer to change the app name to `common-tekton-manifests-int`. As I mentioned before, we need to avoid naming conflicts.